app/javascript/dashboard/components/widgets/conversation/conversationCardComponents/CardLabels.vue

```diff
@@ -2,10 +2,11 @@
   <div
     v-show="activeLabels.length"
```

Comment:
Ideally this top level `v-show` is not good, but it relies heavily on `activeLabels`, I've added another comment about it, feel free to resolve this